test/mock/mock.jwks.service.ts

```diff
@@ -7,7 +7,6 @@ export class MockJwksService implements IJwksService {
 
     private readonly certSigningKey: CertSigningKey = {
         kid: 'abcd',
-        nbf: 'efgh',
```

Comment:
just wanted to raise awareness, just in case:  was this line meant to be deleted?